src/views/VenuesListWithFilter/FilterList/Neighbourhoods/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import ListItem from '@mui/material/ListItem'
+import Checkbox from '@mui/material/Checkbox'
+
+export function Neighbourhoods ({ checkedNeighbourhoods, setCheckedNeighbourhoods }) {
+  const neighbourhoodsFilters = [
+    'lake',
+    'forest',
+    'mountains',
+    'sea',
+    'river',
+    'park',
+    'zoo',
+    'national park'
+  ]
+
+  const handleToggle = (value) => () => {
```

Comment:
I see multiple components that looks similar… `RoomAmenities`, `Neighbourhoods` and `Amenities`. All of them have the same function and have very similar template. Maybe there can be a new component introduced? The component with all of the code that is repeating. It could be used in all of the 3 components. WDYT?